app/src/main/res/values/strings.xml

```diff
@@ -46,9 +46,9 @@
     <string name="message_welcome">Welcome to WADBS, a small companion app that makes your life as a developer much easier! Before you start, please, grant some necessary permissions.</string>
     <string name="message_explain_notifications">WADBS will display a notification when it encounters an error which is usually because it lacks a needed set of privileges.</string>
     <string name="message_explain_root_access">Root access allows WADBS to change the state of Wireless Debugging, gather connection data and interact with KDE Connect to instantly pair your Android device with your PC via local network.\nThis permission is optional if you use Shizuku.</string>
-    <string name="message_explain_shizuku">Alternatively to Root access WADBS can utilize Shizuku to gain the needed set of privileges. Note that KDE Connect integration does not work with Shizuku.</string>
+    <string name="message_explain_shizuku">Alternatively to root access WADBS can utilize Shizuku to gain the needed set of privileges. Note that KDE Connect integration does not work with Shizuku.</string>
     <string name="message_grant_permission_before_leaving">Grant all of the required permissions and then tap \"Continue\".</string>
-    <string name="message_disable_magisk_notification">Magisk displays a toast whenever some app is granted root access. WADBS frequently checks the state of Wireless Debugging on your device to update its widgets (you can edit the interval in settings). This makes Magisk\'s notifications somewhat annoying.\n\nYou can disable Magisk\'s root access notifications from the \"Superuser\" tab (tap on WADBS in the list there to see more options for it).</string>
+    <string name="message_disable_magisk_notification">Magisk displays a toast whenever some app is granted root access. WADBS frequently checks the state of Wireless Debugging on your device to update its widgets (you can edit the interval in Settings). This makes Magisk\'s notifications somewhat annoying.\n\nYou can disable Magisk\'s root access notifications from the \"Superuser\" tab (tap on WADBS in the list there to see more options for it).</string>
```

Comment:
"Settings" with capital S may be thought as referring to system settings. Here we are referring to settings inside of WADBS